### ISSUE TITLE
Mail attribute typo: excerpt

### ISSUE THREAD

**Issue by rkubik** [ncjones/python-guerrillamail#3]
Typo crashes my program, I expect `excerpt` but get `exerpt`.

**Comment by ncjones** [comment]
Can you please provide steps to reproduce the issue?

**Comment by rkubik** [comment]
Due to a typo the Mail.exerpt attribute is never set. In the `from_response` method the `response_data` object returned by the API will contain a JSON attribute `mail_excerpt`. You can see that you try and find `mail_exerpt`, a typo that causes the attribute to never be set.

Reproduce with:

```
from guerrillamail import GuerrillaMailSession
session = GuerrillaMailSession()
print session.get_session_state()['email_address']
print 'BODY'
print session.get_email(1).body
# Should be the same as body, but because of a typo it will always be None
print 'EXCERPT'
print session.get_email(1).exerpt
```

**Comment by ncjones** [comment]
@rkubik 

This is fixed in 73f4b0a and released as version 0.1.2.

Your other issues with "mail_excerpt" and "mail_body" look like server-side issues. Please log a separate issue if you think this is a client-side issue.

**Comment by rkubik** [comment]
Will do, thank you.
